### ISSUE TITLE
Update expected profile occurrance terms to RFC 2119

### ISSUE THREAD

**Issue by marlontaylor** [STIXProject/stix-validator#61]
Currently:
```python
# Occurrence values
OCCURRENCE_PROHIBITED = 'prohibited'
OCCURRENCE_REQUIRED = 'required'
OCCURRENCE_OPTIONAL = 'optional'
OCCURRENCE_SUGGESTED = 'suggested'
ALLOWED_OCCURRENCES = (
OCCURRENCE_OPTIONAL,
OCCURRENCE_PROHIBITED,
OCCURRENCE_REQUIRED,
OCCURRENCE_SUGGESTED
)
```

Should be updated to  RFC 2119 (see below) or dynamically read from profile
- MUST
- SHOULD
- MAY
- SHOULD NOT
- MUST NOT

**Comment by bworrell** [comment]
This is being worked on in the [issue_61](https://github.com/STIXProject/stix-validator/tree/issue_61) branch and potentially resolved in 0c7a3c9103dc1658a29e511c16ac920b43e3f4d4. The old profile behaves the same, but I need to test against the new terms.

Sorry to @bschmoker... I busted #32 when I pushed stix-validator v2.0 :(

**Comment by marlontaylor** [comment]
I changed the terms within the profile from RFC2119 to their legacy equivalent and the validator ran without error.

**Comment by bworrell** [comment]
Fixed in #62
